exercises/resistor-color-duo/.meta/hints.md

```diff
@@ -8,3 +8,32 @@ value :: (Color, Color) -> Int
 
 that only accepts two colors and produces the resistor's numeric value
 component of its resistance.
+
+**Bonus - Property Testing:**
+
+This exercise is a good opportunity to write some [property tests](https://en.wikipedia.org/wiki/Property_testing). If you haven't written any before here are some resources:
+
+1. [School of Haskell](https://www.schoolofhaskell.com/user/pbv/an-introduction-to-quickcheck-testing).
+2. [Real world Haskell](http://book.realworldhaskell.org/read/testing-and-quality-assurance.html)
```

Comment:
Wow, a chapter in RWH that isn't outdated! QuickCheck is surprisingly much like it used to be.

```suggestion
1. School of Haskell: [An introduction to QuickCheck testing](https://www.schoolofhaskell.com/user/pbv/an-introduction-to-quickcheck-testing)
2. Real World Haskell: [Chapter 11. Testing and quality assurance](http://book.realworldhaskell.org/read/testing-and-quality-assurance.html)
```